eng/build-job.yml

```diff
@@ -123,10 +123,10 @@ jobs:
 
     # Build
     - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
-      - script: ./build.sh $(buildConfig) $(archType) $(crossArg) -skiptests -skipnuget $(clangArg) $(stripSymbolsArg) $(officialBuildIdArg) /p:ContinuousIntegrationBuild=true
+      - script: ./build.sh $(buildConfig) $(archType) $(crossArg) -ci -skiptests -skipnuget $(clangArg) $(stripSymbolsArg) $(officialBuildIdArg) /p:ContinuousIntegrationBuild=true
```

Comment:
Do we need to explicitly pass  `/p:ContinuousIntegrationBuild=true` ? I though that `-ci` if used on `/eng/common/msbuild.sh` would set this property for us?

Reply:
Oh yeah, that was from before we were going through Arcade scripts and I missed removing it when switching to using the Arcade scripts.